packages/vscode-zipfs/rollup.config.js

```diff
@@ -0,0 +1,27 @@
+import cjs     from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import path    from 'path';
+import esbuild from 'rollup-plugin-esbuild';
+
+const mode = process.env.NODE_ENV || `production`;
+
+// eslint-disable-next-line arca/no-default-export
+export default async () => ({
+  input: `./sources/index.ts`,
+  output: {
+    file: `./build/index.js`,
+    format: `cjs`,
+  },
+  plugins: [
+    resolve({
+      extensions: [`.mjs`, `.js`, `.ts`, `.tsx`, `.json`],
+      rootDir: path.join(__dirname, `../../`),
+      jail: path.join(__dirname, `../../`),
+      preferBuiltins: true,
+    }),
+    esbuild({tsconfig: false, target: `node12`, format: `cjs`}),
+    cjs({transformMixedEsModules: true, extensions: [`.js`, `.ts`]}),
+    mode === `production` && (await import(`rollup-plugin-terser`)).terser({ecma: 2019}),
```

Comment:
Does the dynamic import really matter?

Reply:
The [rollup docs](https://rollupjs.org/guide/en/#plugins) have an example using `rollup-plugin-terser` where they state:
> This example also demonstrates how to use an async IIFE and dynamic imports to avoid unnecessary module loading, which can be surprisingly slow.